### PR TITLE
remove previous incorrect pull request

### PR DESCRIPTION
**Pull request by Alex-Holborn**
re-add prac_06 files, add prac_07 files

### CODE REVIEW

File: prac_07/guitar.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import datetime
+
+
+class Guitar:
+    def __init__(self, name="", year_made=0, cost=0):
```
Comment (lindsaymarkward) [inline prac_07/guitar.py]: cost should be a float, 0.0

File: prac_07/guitar.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        self.name = name
+        self.year_made = year_made
+        self.cost = cost
+        self.current_year = datetime.datetime.now().year
```
Comment (lindsaymarkward) [inline prac_07/guitar.py]: 👍

File: prac_07/guitar_test.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+from prac_07.guitar import Guitar
```
Comment (lindsaymarkward) [inline prac_07/guitar_test.py]: Where are the rest of the tests, including the "expected" outputs?

File: prac_07/guitars.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+from prac_07.guitar import Guitar
+
+guitars = []
+is_finished_inputting = False
```
Comment (lindsaymarkward) [inline prac_07/guitars.py]: No. Please re-do this using the pattern you have been shown. Mention me when you've corrected this.

File: prac_07/guitars.py [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+while not is_finished_inputting:
+    guitar_name = input("Name:")
+    if guitar_name != "":
```
Comment (lindsaymarkward) [inline prac_07/guitars.py]: Read chapter 2 (week 1) lecture notes that say if you need an if to get out of your while loop, that should probably be your condition... This should be your while condition.

File: prac_07/guitars.py [hunk 6]
```diff
@@ -1,27 +1,25 @@
 from prac_07.guitar import Guitar
 
 guitars = []
-is_finished_inputting = False
-while not is_finished_inputting:
+
+guitar_name = input("Name:")
```
Comment (lindsaymarkward) [inline prac_07/guitars.py]: Good. That's nicer.

**Comment by lindsaymarkward** [comment]
Hi @Alex-Holborn - You still need to complete the tests for guitar. Please mention me when you've done these.
This should help with your tests for the assignment classes.

**Comment by Alex-Holborn** [comment]
@lindsaymarkward I got a bit confused by the tests, is this what you were looking for?

File: prac_07/guitar_test.py [hunk 7]
```diff
@@ -1,6 +1,11 @@
...
-new_guitar = Guitar("New Guitar", 1962, 100)
-print(new_guitar)
-print(new_guitar.get_age())
-print(new_guitar.is_vintage())
+current_year = datetime.datetime.now().year
+guitars = []
+guitars.append(Guitar("New Guitar", 1962, 100))
+guitars.append(Guitar("A very old guitar", 1888, 2000.5))
+
+for guitar in guitars:
```
Comment (lindsaymarkward) [inline prac_07/guitar_test.py]: You can't do this in a loop, since the "expected" values should be literals, and will be different for each object.

**Comment by lindsaymarkward** [comment]
See the prac where it says, 
> Do you see how this works? We print our own literal for what we expect if it's right (e.g. 95), then we print what the actual method returns and we look at the output to see if they match. This form of testing is quite 'manual' since we need to read the output and compare it ourselves, but it is a good start.

So for each test, you print the literal expected value, then call the method to see if it matches.

**Pull request merged**